<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>696</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="baseSize">
   <size>
    <width>70</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ion Chamber Voltmeters</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Dark Current</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QueueButton" name="dark_current_button">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Record</string>
        </property>
        <property name="alarmSensitiveContent" stdset="0">
         <bool>false</bool>
        </property>
        <property name="alarmSensitiveBorder" stdset="0">
         <bool>false</bool>
        </property>
        <property name="PyDMToolTip" stdset="0">
         <string>${SCALER}_offset_start.PROC</string>
        </property>
        <property name="channel" stdset="0">
         <string>${SCALER}_offset_start.PROC</string>
        </property>
        <property name="passwordProtected" stdset="0">
         <bool>false</bool>
        </property>
        <property name="password" stdset="0">
         <string/>
        </property>
        <property name="protectedPassword" stdset="0">
         <string/>
        </property>
        <property name="showConfirmDialog" stdset="0">
         <bool>true</bool>
        </property>
        <property name="confirmMessage" stdset="0">
         <string>Is the shutter closed?</string>
        </property>
        <property name="pressValue" stdset="0">
         <string>1</string>
        </property>
        <property name="releaseValue" stdset="0">
         <string>None</string>
        </property>
        <property name="relativeChange" stdset="0">
         <bool>false</bool>
        </property>
        <property name="writeWhenRelease" stdset="0">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="shutter_checkbox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Close the experimental experimental hutch shutter prior to recording dark current. &lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;This feature is disabled pending availability of shutter controls.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Close Shutter</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="shutter_combobox">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="voltmeters_group_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Ion Chambers</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="voltmeters_layout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>If you can read this, something went wrong.</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Auto-Gain</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QueueButton" name="auto_gain_button">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="toolTip">
         <string>Queue a plan to automatically set the selected ion chamber gains to within the range 0.5V to 4.5V.</string>
        </property>
        <property name="text">
         <string>Auto-gain</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Min (V):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="volts_min_line_edit">
        <property name="toolTip">
         <string>The lower threshold (in volts) that the auto-gain plan will use for deciding when the ion chambers' pre-amp gain settings are correct.</string>
        </property>
        <property name="placeholderText">
         <string>E.g. 1.0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Max (V):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="volts_max_line_edit">
        <property name="toolTip">
         <string>The upper threshold (in volts) that the auto-gain plan will use for deciding when the ion chambers' pre-amp gain settings are correct.</string>
        </property>
        <property name="placeholderText">
         <string>E.g. 4.5</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QueueButton</class>
   <extends>QPushButton</extends>
   <header>firefly.queue_button</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>shutter_checkbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>shutter_combobox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
